src/main/java/org/cyclonedx/gradle/utils/CycloneDxUtils.java

```diff
@@ -4,6 +4,8 @@
 
 public class CycloneDxUtils {
 
+    public static final CycloneDxSchema.Version DEFAULT_SCHEMA_VERSION = CycloneDxSchema.Version.VERSION_15;
```

Comment:
Centralising the default version here should reduce the likelihood to change the default version in the Task property without also updating this util class, preventing the current bug to happen again.

It's also possible to configure the default directly to `CycloneDxSchema.VERSION_LATEST`, but it would be less obvious which version is that just by reading the code, so I opted for a more explicit configuration. Thoughts?